tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -327,6 +328,44 @@ public void AcquireTokenSameUserTest()
             }
         }
 
+        [TestMethod]
+        [DeploymentItem(@"Resources\CustomInstanceMetadata.json")]
+        public async Task AcquireTokenInterative_WithCustomInstanceMetadata_Async()
+        {
+            string instanceMetadataJson = File.ReadAllText(
+                ResourceHelper.GetTestResourceRelativePath("CustomInstanceMetadata.json"));
+
+            using (var harness = CreateTestHarness())
+            {
+                // No instance discovery is made - it is important to not have this mock handler added
+                // harness.HttpManager.AddInstanceDiscoveryMockHandler();
+
+                PublicClientApplication app = PublicClientApplicationBuilder.Create(MsalTestConstants.ClientId)

```

Comment:
nit: these read better in code reviews if they're wrapped like on line 356 (4-space tab indented next line instead of aligned under .Create)